docs/customization/docker.rst

```diff
@@ -47,7 +47,7 @@ Usage
 
    .. code-block:: bash
 
-    $ docker-compose up
+    $ docker-compose up -d
```

Comment:
any opinions on this ?

Reply:
> any opinions on this `-d` ?

Don't you think it's likely to create issue reports saying "nothing happened :-S"?

Reply:
that depends on whether you are using it for dev or production purposes.

Personally, I like to do `docker-compose up -d`

and then if I want to see logs,

```docker-compose logs -f`

Reply:
Agreed and me too but the doc is for development so not detaching might be more intuitive for beginners, don't you think?

Reply:
@stephenmoloney I'd love to merge this but I also feel that for local development attached mode is more useful.

Reply:
sure, I'll remove `-d`